proxy-node-shared/src/test/java/uk/gov/ida/notification/contracts/verifyserviceprovider/AttributesBuilder.java

```diff
@@ -1,59 +1,64 @@
 package uk.gov.ida.notification.contracts.verifyserviceprovider;
 
-import org.joda.time.DateTime;
-import org.joda.time.chrono.ISOChronology;
+import uk.gov.ida.saml.core.domain.Gender;
+import uk.gov.ida.verifyserviceprovider.dto.NonMatchingAddress;
+import uk.gov.ida.verifyserviceprovider.dto.NonMatchingAttributes;
+import uk.gov.ida.verifyserviceprovider.dto.NonMatchingVerifiableAttribute;
+import uk.gov.ida.verifyserviceprovider.dto.NonMatchingTransliterableAttribute;
 
+
+import java.time.LocalDate;
 import java.util.ArrayList;
 import java.util.List;
 
 import static java.util.Collections.singletonList;
+import static uk.gov.ida.saml.core.domain.Gender.NOT_SPECIFIED;
 
 public class AttributesBuilder {
 
     public static final String FIRST_NAME = "Jean";
     public static final String MIDDLE_NAME = "Paul";
     public static final String LAST_NAME = "Smith";
-    public static final String GENDER = "NOT_SPECIFIED";
-    public static final DateTime VALID_FROM = createDateTime(2001, 1, 1, 12, 0);
-    public static final DateTime DATE_OF_BIRTH = createDateTime(1990, 1, 1, 0, 0);
-
-    private List<Attribute<String>> firstNames = new ArrayList<>(singletonList(createAttribute(FIRST_NAME)));
-    private List<Attribute<String>> middleNames = new ArrayList<>(singletonList(createAttribute(MIDDLE_NAME)));
-    private List<Attribute<String>> lastNames = new ArrayList<>(singletonList(createAttribute(LAST_NAME)));
-    private Attribute<String> gender = createAttribute(GENDER);
-    private List<Attribute<DateTime>> datesOfBirth = new ArrayList<>(singletonList(createAttribute(DATE_OF_BIRTH)));
-
-    public Attributes build() {
-        return new Attributes(
+    public static final LocalDate VALID_FROM = createDateTime(2001, 1, 1);
+    public static final LocalDate DATE_OF_BIRTH = createDateTime(1990, 1, 1);
+
+    private List<NonMatchingTransliterableAttribute> firstNames = new ArrayList<NonMatchingTransliterableAttribute>(singletonList(createNonMatchingTransliterableAttribute(FIRST_NAME)));
```

Comment:
I think you can get away with dropping the new ArrayList<...>(...) and just use the singletonList(...) utility method which will give you a List of the right type anyway for this and the other instances.

Reply:
Intellij was hiding it from me, will remove.